org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/codegen/BranchLabel.java

```diff
@@ -28,7 +28,7 @@ public class BranchLabel extends Label {
 	protected int targetStackDepth = -1;
 	public final static int WIDE = 1;
 	public final static int USED = 2;
-	private OperandStack switchSaveTypeBindings;
+	private OperandStack operandStack;
 
```

Comment:
This is just a rename refactoring, to align the field name with what it is called elsewhere.